### PR TITLE
Add blog about BSP in sbt

### PR DESCRIPTION
**Pull request by adpi2**

### CODE REVIEW

**Comment by SethTisue** [comment]
I usually make this same suggestion on everybody's blog post drafts:

I'd suggest leading with the most important material. For two reasons: so someone can tell early whether they want to keep reading; and so someone gets the main takeaways even if they bail early.

The most important material always answers the question: why should I, the reader, care? At the start, the reader doesn't care what BSP is, or who was involved in a collaboration on it, or when that collaboration started. And they don't know — yet! — whether they care to settlle in and read a history of BSP. You haven't given them a reason to care yet. All of that material should come later.

Why should the reader care? You're the author, you know that best, but I guess it goes something like: we want Scala to have good IDE support and editor support. BSP enables that. Metals and IntelliJ already speak BSP, but there's this limitation and that limitation because sbt didn't speak BSP yet. Now that sbt speaks BSP, users will see the following improvements that will help them get their work done.

File: blog/_posts/2020-10-20-bsp-in-sbt.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+
+IntelliJ Idea is also a build client, but it does not rely on a language server. It interacts directly with the build server to import the project and perform compilation on save. The Scala language support in IntelliJ is embedded in the Scala plugin.
+
+## What's new
+
+With the recent built-in support of BSP in sbt, it is going to be possible for Metals and IntelliJ to connect to the sbt server and communicate with it directly. This solution is an alternative to the current status quo of using Bloop as a third-party BSP server.
+
+Bloop still offers many advantages compared to sbt server. It can serve several build clients, on different projects, and run the requests concurrently. It also supports DAP, the Debug Adapter Protocol, which provides code editors with the ability to debug applications and evaluate code at runtime.
+
+Choosing Bloop or sbt as the build server depends on the project you are working on and the developer experience you are looking for. In the following paragraph we describe the main characteristics of using sbt as a build server.
```
Comment (martijnhoekstra) [inline blog/_posts/2020-10-20-bsp-in-sbt.md:38]: It would be nice to contrast the below to what you get when you use bloop as a build server.

**Comment by adpi2** [comment]
Thanks @SethTisue and @martijnhoekstra  for your suggestions. I pushed an update of the blog post.

**Comment by adpi2** [comment]
@sjrd Could you review the last commit before merging?

File: blog/_posts/2020-10-27-bsp-in-sbt.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+layout: blog-detail
+post-type: blog
+by: Adrien Piquerez, Scala Center
+title: BSP Support in sbt 1.4
+---
+
+sbt is a prominent tool of the Scala ecosystem. Yet it is poorly integrated in our IDEs and code editors, which are used to relying on internals or third parties to perform operations like compiling, running and testing.
+
+From this lack of integration derives a number of inconveniences:
+- Duplicated compilation times
```
Comment (sjrd) [inline blog/_posts/2020-10-27-bsp-in-sbt.md:11]:
```suggestion
- Duplicated compilation times between sbt and the IDE
```

File: blog/_posts/2020-10-27-bsp-in-sbt.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+- False compiler errors on the IDE
+- Outdated dependencies causing runtime failures
```
Comment (sjrd) [inline blog/_posts/2020-10-27-bsp-in-sbt.md:13]:
```suggestion
- Outdated dependencies causing run-time failures
```

**Review by sjrd — APPROVED** [review]
Two suggestions for the intro.

**Pull request merged**